server/src/main/java/org/opensearch/index/shard/IndexShard.java

```diff
@@ -305,6 +305,9 @@ Runnable getGlobalCheckpointSyncer() {
 
     private final CheckpointRefreshListener checkpointRefreshListener;
 
+    private volatile ReplicationCheckpoint latestUnprocessedCheckpoint;
+    private volatile long latestSeqNo;
```

Comment:
This is a hack until https://github.com/opensearch-project/OpenSearch/pull/2576 is merged to correctly track the local cp on replicas.

Reply:
#2576 has been merged, so you can rebase and remove this part